app/models/event.rb

```diff
@@ -154,6 +154,15 @@ def compute_occupied_places
     application_letters.where(status: ApplicationLetter.statuses[:accepted]).count
   end
 
+  # locks the ability to change application statuses
+  def lock_application_status
+    update(application_status_locked: true)
+  end
+  # unlocks the ability to change application statuses
+  def unlock_application_status
```

Comment:
I see what you mean, but I'm not sure if our PO really had this in mind. Since when you unlock it again, everything can be changed (not only "Nachrücker durch Absagen"), Further this causes the send accept/reject mail buttons to appear again.